### ISSUE TITLE
MacOS only?

### ISSUE THREAD

**Issue by ReconCoyote978** [rayriffy/pixiv-bookmarks-visualizer#4]
Hi, I am trying to run this program on Microsoft Windows 11, but the OS does not support the file types of the scripts, and the information on readme is unclear how to start using the program. Is it because the program was not designed for Windows at all?

**Comment by rayriffy** [comment]
I don't think so. Could you describe of which script are you running in each setup step in order? Also together with error stack so I could take a look.

But regardless, Windows filesystem is completely different than UNIX-based system and very problematic for Node filesystem. I recommended you to run in WSL instead for easier time.

**Comment by rayriffy** [comment]
Closing due to inactivity
